packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -300,6 +299,33 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
       value: trigger,
     };
 
+    const editorComponent = ({ isEditable }) => {
```

Comment:
This could be present after line 375

Reply:
My latest commit code is working , I just need to edit the nit picks.